STM32Cube/Core/Src/ICM-20948.c

```diff
@@ -0,0 +1,180 @@
+#include <stdint.h>
+#include <stdbool.h>
+#include "my2c.h"
+#include "ICM-20948.h"
+#include "ICM-20948_regmap.h"
+#include "cmsis_os.h"
+
+//I2C addresses
+static uint8_t ICM_20948_addr;
+static uint8_t AK09916_mag_addr;
+
+// This driver assumes that I2C is already initialized
+bool ICM_20948_init() {
+    ICM_20948_addr = 0x69;
```

Comment:
There are defines in `ICM-20948.h` for the device addresses, however they are specified as literals here. Does it make sense to use the defines everywhere for these values?